test/components/connectors/test_openapi_service.py

```diff
@@ -88,3 +129,20 @@ def test_for_internal_raw_data_field(self):
             "openapi3 changed. Model should have a _raw_data field, we rely on it in OpenAPIServiceConnector"
             " to get the raw data from the service response"
         )
+
+    @pytest.mark.integration
+    @pytest.mark.skipif(not os.getenv("GITHUB_TOKEN"), reason="GITHUB_TOKEN is not set")
+    def test_run(self, genuine_fc_message):
+        openapi_service = OpenAPIServiceConnector()
+        github_compare_schema = requests.get("https://bit.ly/github_compare").json()
```

Comment:
Don't do this please. If we need some json schema let's add it as a file or a string somewhere.